backend/de.metas.edi/src/main/java-gen/de/metas/esb/edi/model/I_EDI_DesadvLine_InOutLine.java

```diff
@@ -0,0 +1,370 @@
+package de.metas.esb.edi.model;
+
+import org.adempiere.model.ModelColumn;
+
+import javax.annotation.Nullable;
+import java.math.BigDecimal;
```

Comment:
why do we need an `m:n`-Table in addition to  `M_InOutLine.EDI_DesadvLine_ID`

Reply:
actually, it's not an m:n, it's a 1:1, I just didn't  want to add the new columns to shipment line